components/browser/errorpages/src/main/res/values-fi/strings.xml

```diff
@@ -20,6 +20,13 @@
     <!-- The document title and heading of the error page shown when a website sends back unusual and incorrect credentials for an SSL certificate. -->
     <string name="mozac_browser_errorpages_security_ssl_title">Suojattu yhteys epäonnistui</string>
 
+    <!-- The error message shown when a website sends back unusual and incorrect credentials for an SSL certificate. -->
+    <string name="mozac_browser_errorpages_security_ssl_message"><![CDATA[¶
+      <ul>¶
+        <li>Avattavaa sivua ei voida näyttää, koska vastaanotetun datan alkuperää ei kyetty varmentamaan.</li>¶
+        <li>Ilmoitathan ongelmasta sivuston omistajalle.</li>¶
+      </ul>¶]]></string>
```

Comment:
@Pike Those `¶` symbols look like they should not be in here? 🤔

Reply:
Probably. That's for @Delphine at this point, though.

Reply:
This fell through the cracks for some reason.
@pocmo: I see you've updated this on Pontoon. We're good now right? Or do you still need something from me?